SparkBuilderGeneratorPlugin/src/com/helospark/spark/builder/handlers/codegenerator/component/helper/TypeDeclarationFromSuperclassExtractor.java

```diff
@@ -51,9 +51,9 @@ public TypeDeclarationFromSuperclassExtractor(CompilationUnitParser compilationU
     private Optional<CompilationUnit> getCompilationUnit(IType superClassType) {
         Optional<CompilationUnit> result = empty();
```

Comment:
Could you use Java 9? Optional.or could be a better solution.

Reply:
I don't want to increase the minimum JRE requirement for the plugin (and Eclipse).

Reply:
I totally understand your point.